stdlib/standardNormal.hk

```diff
@@ -23,3 +23,11 @@ def stdCauchy():
 def cauchy(a real, alpha prob):
 	X <~ stdCauchy()
 	return a + alpha*X
+
+# implementation of hyperbolic secant distribution as a transformation
+# of standard Cauchy distribution
+def hyperbolicSecant():
+	x <~ stdCauchy()
+	return log(abs(x/pi()))
```

Comment:
Are you sure that that is how pi works? I thought it was just a constant, no need for the ().